packages/itwinui-react/package.json

```diff
@@ -106,17 +106,17 @@
     "@floating-ui/react": "^0.26.23",
     "@itwin/itwinui-illustrations-react": "^2.1.0",
     "@swc/helpers": "^0.5.11",
-    "@tanstack/react-virtual": "^3.8.2",
+    "@tanstack/react-virtual": "^3.12.0",
```

Comment:
We should have a changeset for this.

Also we should probably bump other deps too, e.g. `@floating-ui/react`, (in a separate PR).